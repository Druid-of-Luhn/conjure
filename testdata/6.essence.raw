Spec
  { language = "Essence"
  , version = [ 2
              , 0
              ]
  , topLevelBindings = []
  , topLevelWheres = []
  , objective = Nothing
  , constraints = [ GenericNode
                      Lt
                      [ ValueInteger 1
                      , ValueInteger 2
                      ]
                  , GenericNode
                      Lt
                      [ ValueInteger 2
                      , ValueInteger 3
                      ]
                  ]
  }