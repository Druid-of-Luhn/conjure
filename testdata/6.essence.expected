language Essence 2.0

such that
    1 < 2,
    2 < 3
